src/core_plugins/kbn_vislib_vis_types/public/controls/gauge_options.js

```diff
@@ -107,6 +107,10 @@ module.directive('gaugeOptions', function () {
         $scope.customColors = true;
       });
 
+      $scope.editorState.requiredText = i18n('kbnVislibVisTypes.params.gauge.requiredText', {
```

Comment:
Please don't set that value on the `editorState`, but use the `$scope` directly. A translated string is NOT part of the editor state.